main/src/ca/uwaterloo/flix/language/phase/jvm/BytecodeInstructions.scala

```diff
@@ -38,6 +39,17 @@ object BytecodeInstructions {
 
     def visitVarInstruction(opcode: Int, v: Int): Unit =
       visitor.visitVarInsn(opcode, v)
+
+    def visitJumpInstruction(opcode: Int, label: Label): Unit =
```

Comment:
What is opcode and should it be an integer?

Reply:
`Opcodes.GOTO` or `Opcodes.IFNULL` etc. Do you want me to introduce enums?

Reply:
Enums or multiple functions. We can do it in the next PR.